config/rollup.ses.config.js

```diff
@@ -0,0 +1,31 @@
+import config from './rollup.iife.config';
+import fs from "fs";
+import { builtinModules as builtin } from "module";
+import jscc from 'rollup-plugin-jscc';
+import replace from "@rollup/plugin-replace";
+
+const pkg = JSON.parse(fs.readFileSync("./package.json"));
+delete pkg.dependencies["ejs"];
```

Comment:
have to remove `ejs`, because this line is not SES compatible: https://github.com/mde/ejs/blob/main/lib/ejs.js#L109